extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -257,6 +266,8 @@ private Map<String, String> prepareConfiguration(
         configProperties.put(APPLICATION_TYPE_CONFIG_KEY, oidcApplicationType);
         configProperties.put(CLIENT_ID_CONFIG_KEY, oidcClientId);
         configProperties.put(CLIENT_SECRET_CONFIG_KEY, oidcClientSecret);
+        configProperties.put(OIDC_USERS, users.entrySet().stream()
```

Comment:
I’m not totally sure it’s a good idea to rely on a toString impl for formatting as this might be subject to change.

Reply:
Hi @gsmet thanks, good point, I did not think about it; I can update this code to convert an entry to string avoiding `toString()` during one of then next updates to `KeycloakDevServicesProcessor`